src/constants/networks.ts

```diff
@@ -1,12 +1,18 @@
 import environments from '../../public/configs/env.json';
 
-// TODO: Update for Mainnet deployment
-export const AVAILABLE_ENVIRONMENTS =
-  import.meta.env.MODE === 'production'
-    ? environments.deployments.TESTNET
-    : environments.deployments.DEV;
+const CURRENT_MODE = ({
+  production: 'MAINNET',
+  testnet: 'TESTNET',
+  staging: 'DEV',
+  development: 'DEV',
+}[import.meta.env.MODE] ?? 'MAINNET') as 'MAINNET' | 'TESTNET' | 'DEV';
```

Comment:
Slightly concerned with leveraging `import.meta.env.MODE`, e.g it has default value of `development ` / `production` for vite commands. I wonder if we should just add a separate ENV VAR